application/testing/CMakeLists.txt

```diff
@@ -211,6 +211,13 @@ else()
   endif()
 endif()
 
+# Needs splat sorting with compute shaders
+if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240201)
```

Comment:
technically should be 0202

Reply:
and you still need https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10862#note_1477632 anyway